vllm/v1/metrics/loggers.py

```diff
@@ -17,6 +17,9 @@
 
 _LOCAL_LOGGING_INTERVAL_SEC = 5.0
 
+STANDARD_LOGGING_LOGGER_NAME = "logging"
+PROMETHEUS_LOGGING_LOGGER_NAME = "prometheus"
```

Comment:
nit: unnecessarily long names
```suggestion
STANDARD_LOGGER_NAME = "logging"
PROMETHEUS_LOGGER_NAME = "prometheus"
```